js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,87 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone();
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+  }
+
+  get events() {
+    return {
+      'change input[name="windowControlStyle"]': 'changedWindowStyle'
+    };
+  }
+
+  changedWindowStyle(event) {
+    app.localSettings.set('windowControlStyle', $(event.target).val()); 
```

Comment:
That's a good point, it would be more consistent to save the local storage data on save instead of when the button is clicked on.